src/test/java/com/google/longrunning/OperationsClientTest.java

```diff
@@ -228,7 +228,7 @@ public void deleteOperationExceptionTest() throws Exception {
       client.deleteOperation(name);
       Assert.fail("No exception raised");
     } catch (ApiException e) {
-      Assert.assertEquals(Status.INTERNAL.getCode(), e.getStatusCode());
+      Assert.assertEquals(Status.INVALID_ARGUMENT.getCode(), e.getStatusCode());
```

Comment:
Potentially. But the intermediate state will not compile, because of the generated `PageResponseWrappers` for longrunning which uses the updated `Page` and `PagedListResponse` interfaces.